code/modules/projectiles/guns/energy/laser_gatling.dm

```diff
@@ -83,12 +84,15 @@
 /obj/item/minigunpack/update_icon_state()
 	icon_state = armed ? "notholstered" : "holstered"
 	return ..()
+/obj/item/minigunpack/proc/drain_power()
```

Comment:
i'm like. 99% sure this isn't needed

Reply:
It will refuse to compile without it but if it's really that bad I could find a way around doing it this way